packages/lib/slugify.test.ts

```diff
@@ -30,6 +30,21 @@ describe("slugify", () => {
     expect(slugify("$hello-there_")).toEqual("hello-there");
   });
 
+  it("should keep periods as is except the start and end", () => {
+    expect(slugify("hello.there")).toEqual("hello.there");
+    expect(slugify("h.e.l.l.o.t.h.e.r.e")).toEqual("h.e.l.l.o.t.h.e.r.e");
```

Comment:
Do we want to allow usernames like this? 🤔

Reply:
<img width="642" alt="Screenshot 2023-10-06 at 12 33 25 PM" src="https://github.com/calcom/cal.com/assets/53316345/8728f80f-8587-4dc6-9363-f45a837b4508">

It was mentioned in the issue requirements.

Reply:
> Do we want to allow usernames like this? 🤔

yes, since it's actually the official spec of email usernames